plugins/modules/ios_static_routes.py

```diff
@@ -1,46 +1,31 @@
 #!/usr/bin/python
-# -*- coding: utf-8 -*-
-# Copyright 2019 Red Hat
-# GNU General Public License v3.0+
-# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
-
-#############################################
-#                WARNING                    #
-#############################################
 #
-# This file is auto generated by the resource
-#   module builder playbook.
+# This file is part of Ansible
 #
-# Do not edit this file manually.
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
 #
-# Changes to this file will be over written
-#   by the resource module builder.
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
 #
-# Changes should be made in the model used to
-#   generate this file or in the resource module
-#   builder template.
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 #
-#############################################
-
 """
 The module file for ios_static_routes
 """
-
-
 from __future__ import absolute_import, division, print_function
 
 __metaclass__ = type
-
-
-ANSIBLE_METADATA = {
-    "metadata_version": "1.1",
-    "status": ["preview"],
-    "supported_by": "network",
-}
-
+ANSIBLE_METADATA = {"metadata_version": "1.1", "supported_by": "Ansible"}
```

Comment:
command in running-config seems to be wrong? 
 command C(show configuration commands | grep 'static route')